### ISSUE TITLE
Custom stick mode setting is not working.

### ISSUE THREAD

**Issue by Abhishek284** [dji-sdk/Mobile-SDK-Android#238]
Device : Mavic pro
SDK: 4.4.1

On trying to set to Custom , the result will be successful but it does not work
On trying to set to mode one or mode two , it works fine.

Please find the attached GitHub project where I am trying to set custom mode,
[Custom Stick Mode demo](https://github.com/Abhishek284/CustomStickDemo)

**Comment by Michael-DJI** [comment]
could pls show your code? are u calling setCustomAircraftMapping?

**Comment by Abhishek284** [comment]
```
setLeftStickHorizontalMapping(AircraftStickMappingTarget.ROLL, false);
        setLeftStickVerticalMapping(AircraftStickMappingTarget.PITCH, false);
        setRightStickHorizontalMapping(AircraftStickMappingTarget.YAW, false);
        setRightStickVerticalMapping(AircraftStickMappingTarget.THROTTLE, false);
```

```
private AircraftStickMapping leftStickHorizontalMapping;
    private AircraftStickMapping leftStickVerticalMapping;
    private AircraftStickMapping rightStickHorizontalMapping;
    private AircraftStickMapping rightStickVerticalMapping;

public void setLeftStickHorizontalMapping(AircraftStickMappingTarget aircraftStickMappingTarget, boolean isReversed) {
        leftStickHorizontalMapping = buildAircraftStickMapping(aircraftStickMappingTarget, isReversed);
    }

    public void setLeftStickVerticalMapping(AircraftStickMappingTarget aircraftStickMappingTarget, boolean isReversed) {
        leftStickVerticalMapping = buildAircraftStickMapping(aircraftStickMappingTarget, isReversed);
    }

    public void setRightStickHorizontalMapping(AircraftStickMappingTarget aircraftStickMappingTarget, boolean isReversed) {
        rightStickHorizontalMapping = buildAircraftStickMapping(aircraftStickMappingTarget, isReversed);
    }

    public void setRightStickVerticalMapping(AircraftStickMappingTarget aircraftStickMappingTarget, boolean isReversed) {
        rightStickVerticalMapping = buildAircraftStickMapping(aircraftStickMappingTarget, isReversed);
    }



    private void setCustomAircraftMapping() {
        Aircraft aircraft = getAircraft();
        if (aircraft != null && aircraft.getRemoteController() != null) {
            aircraft.getRemoteController().setAircraftMappingStyle(AircraftMappingStyle.STYLE_CUSTOM, new CommonCallbacks.CompletionCallback() {
                @Override
                public void onResult(DJIError djiError) {
                    if (djiError == null) {
                        setNewAircraftStickMapping();
                    }
                }
            });
        }
    }
    
    public void setNewAircraftStickMapping() {
        final Aircraft aircraft = getAircraft();
        final AircraftMapping aircraftMapping = new AircraftMapping(leftStickVerticalMapping, leftStickHorizontalMapping, rightStickVerticalMapping, rightStickHorizontalMapping);
        aircraftMapping.aircraftMappingStyle = AircraftMappingStyle.STYLE_CUSTOM;

        if (aircraft != null && aircraft.getRemoteController() != null) {
            aircraft.getRemoteController().setCustomAircraftMapping(aircraftMapping, new CommonCallbacks.CompletionCallback() {
                @Override
                public void onResult(final DJIError djiError) {
                    mHandler.post(new Runnable() {
                        @Override
                        public void run() {
                            if (djiError == null) {
                                showMessage("Successfully set to custom");

                            } else {
                                showMessage(djiError.getDescription());
                            }
                        }
                    });
                }
            });
        }
    }

    private AircraftStickMapping buildAircraftStickMapping(AircraftStickMappingTarget aircraftStickMappingTarget, boolean isReversed) {
        AircraftStickMapping.Builder aircraftStickMappingBuilder = new AircraftStickMapping.Builder();
        aircraftStickMappingBuilder.mappingTarget(aircraftStickMappingTarget);
        aircraftStickMappingBuilder.isReversed(isReversed);
        return aircraftStickMappingBuilder.build();
    }
```

I am calling setCustomAircraftMapping() method after setting all stick mapping as shown at the top.

After custom is set , right stick horizontal mapping is set to throttle but I had mapped it to AircraftStickMappingTarget.YAW. Others mappings also do not work, they do not move the aircraft at all once the custom mapping is set.

**Comment by Michael-DJI** [comment]
did you get the toast "Successfully set to custom" ?

**Comment by Abhishek284** [comment]
Yes, I got the message "Successfully set to custom".

**Comment by Michael-DJI** [comment]
@Abhishek284 thanks for your feedback, we will fix this issue in next version.

**Comment by Michael-DJI** [comment]
@Abhishek284 did you fix your issue with new version? 
pls report the issue to dev@dji.com it's the official channel to request DJI SDK support now.

**Comment by Stevenhua03** [comment]
Can anyone confirm if this was fixed in one of the updates since version 4.4.1? I checked the release notes and didn't find any information regarding the custom stick mappings.

I'm experiencing the exact same problem as Abhishek284 did with setting a custom style for the sticks. The right stick horizontal mapping gets set to throttle/pitch after I set the style to custom and set my own custom mapping.
